Gaming The Syllabus/Assets/Scripts/IInteractable.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IInteractable
+{
+    public int InteractionPriority { get; }
+
+    public void Interact();
+}
```

Comment:
Great choice on making this an interface. Should make further development much easier.